wp-admin/css/colors/_admin.scss

```diff
@@ -83,7 +83,8 @@ input[type=radio]:checked:before {
 
 /* List tables */
 
-.wrap .add-new-h2:hover,
+.wrap .add-new-h2:hover, /* deprecated */
```

Comment:
Avoid using id selectors<br>Selector should have depth of applicability no greater than 2, but was 3